README.md

```diff
@@ -5,7 +5,9 @@
 For what it is capable of, take a look at [kepler.gl demo app](https://uber.github.io/kepler.gl/#/demo).
 
 Kepler.gl is a redux component that uses redux reducer to store and manage state transitions.
-This package consists of a reducer and the UI components to render and customize the map.
+This package consists of a reducer and the UI components to render and customize the map. 
+
+For information on how to save the map state you've created and have it persist after you've closed the browser, take a look at this tutorial on [vis.academy](http://vis.academy/#/kepler.gl/setup).
```

Comment:
How about linking the text `this tutorial on vis.academy` rather than only `vis.academy`? Otherwise, it looks like the exact tutorial wasn't linked.

Also, you can link directly to `http://vis.academy/#/kepler.gl/`